draft-ietf-tls-esni.md

```diff
@@ -528,16 +529,16 @@ The client then generates the HPKE encryption context and computes the
 encapsulated key, context, and payload as:
 
 ~~~
-    pkR = Deserialize(ECHConfig.public_key)
+    pkR = Deserialize(ECHConfig.contents.public_key)
     enc, context = SetupBaseS(pkR,
                               "tls ech" || 0x00 || ECHConfig)
     payload = context.Seal(ClientHelloOuterAAD,
                            EncodedClientHelloInner)
 ~~~
 
 Note that the HPKE functions Deserialize and SetupBaseS are those which match
-`ECHConfig.kem_id` and the AEAD/KDF used with `context` are those which match
-the client's chosen preference from `ECHConfig.cipher_suites`. The `info`
+`ECHConfig.contents.kem_id` and the AEAD/KDF used with `context` are those which match
```

Comment:
This line and the one below it are longer than 80 characters. (Sorry if this is annoying ... lines that are longer than 80 characters cause `make` to fail. In fact, you should be getting emails from CI telling you that the build is failing.)

Reply:
Aha, I hadn't run make in a few commits, my bad. Fixed now.